README.md

```diff
@@ -46,125 +46,93 @@ Please explore the code to see existing probes and actions.
 
-Configuration values for the Chaos Toolkit Extension for Azure can come from several sources:
+The extension will first try to load the configuration from the `experiment file`. If configuration is not provided in the `Experiment file`, it will try to load it from the `Azure credential file`.
 
-- Experiment file
-- Azure credential file
+[azure-python-sdk]: https://github.com/Azure/azure-sdk-for-python
 
-The extension will first try to load the configuration from the `experiment file`. If configuration is not provided in the `experiment file`, it will try to load it from the `Azure credential file`.
+### Configuration in the `Experiment file`
 
-[creds]: https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-connect-to-secure-cluster
-[requests]: http://docs.python-requests.org/en/master/
-[sdk]: https://github.com/Azure/azure-sdk-for-python
-
-### Credentials
-
-- Secrets in the Experiment file
-
-  ```json
-  {
-    "secrets": {
-      "azure": {
-        "client_id": "your-super-secret-client-id",
-        "client_secret": "your-even-more-super-secret-client-secret",
-        "tenant_id": "your-tenant-id"
-      }
+Set `Tenant ID`, `Client ID` and `Client Secret` in the `Experiment file` as follow:
+```json
+{
+  "secrets": {
+    "azure": {
+      "client_id": "your-super-secret-client-id",
+      "client_secret": "your-even-more-super-secret-client-secret",
+      "tenant_id": "your-tenant-id"
     }
   }
-  ```
-
-  You can retrieve secretes as well from [environment][env_secrets] or [HashiCorp vault][vault_secrets]. 
-
-  
-  If you are not working with Public Global Azure, e.g. China Cloud You can set the cloud environment.
+}
+```
 
-  ```json
-  {
-    "client_id": "your-super-secret-client-id",
-    "client_secret": "your-even-more-super-secret-client-secret",
-    "tenant_id": "your-tenant-id",
-    "azure_cloud": "AZURE_CHINA_CLOUD"
+Additionally you need to provide your `Subscription ID` as follow:
+```json
+{
+  "configuration": {
+    "azure_subscription_id": "your-azure-subscription-id"
   }
-  ```
-
-  Available cloud names:
-
-  - AZURE_CHINA_CLOUD
-  - AZURE_GERMAN_CLOUD
-  - AZURE_PUBLIC_CLOUD
-  - AZURE_US_GOV_CLOUD
-
-  [vault_secrets]: https://docs.chaostoolkit.org/reference/api/experiment/#vault-secrets
-  [env_secrets]: https://docs.chaostoolkit.org/reference/api/experiment/#environment-secrets
-
-
-- Secrets in the Azure credential file
-
-  You can retrieve a credentials file with your subscription ID already in place by signing in to Azure using the az login command followed by the az ad sp create-for-rbac command
-
-  ```bash
-  az login
-  az ad sp create-for-rbac --sdk-auth > credentials.json
-  ```
-
-  credentials.json:
+}
+```
 
-  ```json
+If you are not working with Public Global Azure, e.g. China Cloud You can set the cloud environment as follow:
```

Comment:
as defined below ... instead of as follow

Reply:
done